src/cljs/athens/electron.cljs

```diff
@@ -361,52 +375,94 @@
                                                    #_[:local-storage/navigate]]}
 
                                      ;; whether first or nth time, update athens pages
-                                     #_{:when       :seen-any-of?
-                                        :events     [:fs/create-new-db :reset-conn]
-                                        :dispatch-n [[:db/retract-athens-pages]
-                                                     [:db/transact-athens-pages]]}
+                                    #_{:when       :seen-any-of?
```

Comment:
@agentydragon Seems like the linter should have fixed this. Any idea if that's something it should have caught, or if it’s possible to automate?

Reply:
I'm also surprised the linter didn't catch this. But unfortunately I don't know. CI seems to have passed. My ideas would be to test this, and if I can reproduce that the linter in fact allows indents like this that I'd see as wrong, then I'd play with linter settings, or worst case submit a bug. Otherwise it might be the CI not working correctly, but I've seen it point out lint issues on other PRs, so it's either been broken recently, or it just accepts both alignments here but does find other issues correctly.